### PR TITLE
feat(plasma-b2c,plasma-web): Add `Card` component

### PR DESCRIPTION
**Pull request by neretin-trike**
<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.74.0-canary.67.2494932460.0
  npm install @salutejs/plasma-web@1.109.0-canary.67.2494932460.0
  # or 
  yarn add @salutejs/plasma-b2c@1.74.0-canary.67.2494932460.0
  yarn add @salutejs/plasma-web@1.109.0-canary.67.2494932460.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Demo canvas app deployed!
  
  http://plasma.sberdevices.ru/demo-canvas-app-pr-67/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr-67/
  ui storybook: http://plasma.sberdevices.ru/pr-67/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr-67/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr-67/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr-67/temple-storybook/

File: packages/plasma-b2c/src/components/Card/Card.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    align-items: center;
+
+    color: ${additional.h40[300]};
+
+    svg {
+        color: ${additional.h40[300]};
+    }
+
+    margin-right: 0.75rem;
+`;
```
Comment (Yeti-or) [inline packages/plasma-b2c/src/components/Card/Card.stories.tsx:75]: 👍   лайк за story

Comment (neretin-trike) [reply packages/plasma-b2c/src/components/Card/Card.stories.tsx:75]: 🚀

**Pull request merged**